### ISSUE TITLE
Telemetry error

### ISSUE THREAD

**Issue by samhoppen** [SCasanova/f1dataR#39]
I get the following error when using the `get_driver_telemetry()` function:

`Error: ModuleNotFoundError: No module named 'fastf1'`

**Comment by pbulsink** [comment]
Presuming you have python available to your R environment (verify by running `reticulate::py_available()` and receiving 'TRUE' as the result), you can make sure that the fastf1 package is installed by running `reticulate::py_install("fastf1")`. If that succeeds, then retrying your telemetry request *should* work.

**Comment by pbulsink** [comment]
So, I recently changed dev devices, and had the same issue pop up. 
You can cause reticulate to use a virtual environment by running (change the first argument to name the venv whatever you want): 
```
reticulate::virtualenv_create('f1dataRenv', packages = 'fastf1')
reticulate::use_virtualenv('f1dataRenv')
```
You might need to restart the R session to get reticulate to use the venv. If so, run that second line *before* calling library(fastf1)

**Comment by pbulsink** [comment]
Hi @samhoppen - any update? If this has been working, or you're still having trouble, let us know.

**Comment by SCasanova** [comment]
Talked to Sam. He apparently no longer has this specific issue

**Comment by pbulsink** [comment]
Should also be resolved by #119 and #126
